index.html

```diff
@@ -1431,6 +1427,20 @@
 We provide a [reference implementation](reference/open_pbr_surface.mtlx) in MaterialX, which is based on the derivation in the previous section and the implementation of Autodesk Standard Surface [#Georgiev2019].
 
 
+Flexibility of implementation
+=============================
+
+We want this material model to be renderable on a wide range of platforms, from offline path tracers all the way to real-time game engines on mobile devices.
+
+The Historical background and objectives section explains how the goal of this specification is to provide an ideal, reference appearance. Complete conformance to the specification is defined as reproducing all the physical light transport effects of that ideal appearance.
+
+However, a number of features can be computationally expensive and therefore impractical to support under certain constraints. Enforcing a specific implementation would tailor the material model to a particular set of constraints, limiting its versatility, which would ultimately make it less useful. For this reason, the specification focuses solely on defining the target appearance. The choice of the final BSDF implementation and its associated trade-off is left entirely to the implementer, allowing them to best suit it to their needs.
```

Comment:
Maybe worth detailing what "constraints" means in this context. Presumably hardware constraints, like CPU/GPU speed and memory? What else, things like choice of graphics API/language?